sklearn/linear_model/tests/ransac_iterations.py

```diff
@@ -0,0 +1,20 @@
+from sklearn import linear_model
+from sklearn.linear_model.ransac import _dynamic_max_trials
+import numpy as np
+
+X =np.linspace(0,100,100)[:,None]
+print X.shape
+Y = 2*X
+Y[80:] = np.random.rand(20,1)
+Y[:80] += np.random.rand(80,1)
+
+for i in range(10):
+    clt_ransac = linear_model.RANSACRegressor(linear_model.LinearRegression(),max_trials=100)
+    clt_ransac.fit(X, Y)
+    yhat_ransac = clt_ransac.predict(X)
+    inlier_mask = clt_ransac.inlier_mask_
+    print '*************************'
+    print 'actual     trials:' , clt_ransac.n_trials_
```

Comment:
It's a statistical algo so it can't be guaranteed that n_trials_ won't be higher,  I'll check <= 21. That will give us a 1e-9 probability of failure